### DISCUSSION TITLE
Safe Exam Browser Chromebooks

### DISCUSSION THREAD

**Discussion by lightnessking** [SafeExamBrowser/seb-mac#235] [General]
Hello everyone,

SEB is also available to run as kiosk application on chromebooks as a chrome app. However, as Google is dropping the support somewhere in 2026, what is the alternative route to have SEB running on chromebooks? Is there any information available on the roadmap on how SEB is going to tackle the chrome app support removal?

Sincerely,

Marcel

**Comment by danschlet** [comment]
There is no official SEB version for ChromeOS, as Chromebooks are not used in Education in our region of Europe and we currently don't have resources to develop and maintain such a version. I'm aware of a Chromebook solution for Inspera Assessment. Is there another SEB-like Chromebook version and do you have a link to it?

To be honest we are also not aware what Google intents to use as replacement for today's ChromeOS apps, will they instead support Android apps? There were some attempts to create a SEB version for Android, but unfortunately it didn't materialize until now.

**Reply by lightnessking** [reply]
Hey dan! Thank you for your reply

Here is some information I found on how to use SEB on chromebooks; http://sccstudents.org/seb_chromebook.html
And there is a page for migration suggestions of Google to which they advice migrating to progressive web apps and/or android apps. PWA allows kioskmode, but drops the functionality to verify if it's running in kioskmode. And android does not support kioskmode.
